components/director/internal/domain/label/repository.go

```diff
@@ -165,6 +167,14 @@ func (r *repository) DeleteAll(ctx context.Context, tenant string, objectType mo
 	return errors.Wrapf(err, "while deleting all Label entities from database for %s %s", objectType, objectID)
 }
 
+func (r *repository) DeleteByKeyNotPattern(ctx context.Context, tenant string, objectType model.LabelableObject, objectID string, labelKeyPattern string) error {
+	return r.deleter.DeleteMany(ctx, tenant, repo.Conditions{
+		repo.NewEqualCondition(labelableObjectField(objectType), objectID),
+		repo.NewEqualCondition(tenantColumn, tenant),
+		repo.NewNotRegexConditionString("key", labelKeyPattern),
```

Comment:
Extract `key` as the tenantColumn constant on the above line for consistency and better comprehension

Reply:
Don't think it will bring any value, as currently the hardcoded SQL queries also use the same way "tenant" or "key" or something else. I think it will bring only the need to search for what this constant is pointing to.